content/200-concepts/200-database-connectors/08-sql-server/index.mdx

```diff
@@ -37,6 +37,22 @@ The following example uses SQL authentication (username and password) with an en
 sqlserver://HOST:PORT;database=DATABASE;user=USER;password=PASSWORD;encrypt=true
 ```
 
+<Admonition type="warning">
+
+Note: If you are using any of the following characters in your connection string, you will need to escape them.
+
+```terminal
+:\=;/[]{}  # these are characters that will need to be escaped
+```
+
+To escape these characters, use curly braces `{}` around each special character. As an example:
+
+```terminal
+sqlserver://HOST:PORT;database=DATABASE;user=MyServer{/}MyUser;password=ThisIsA{:}SecurePassword{;};encrypt=true
+```
```

Comment:
🤔

Would adding curly braces around a property's value also work when escaping special characters? Would that be a more convenient way of escaping special characters? e.g. 

```
sqlserver://HOST:PORT;database=DATABASE;user={MyServer/};MyUser;password={ThisIsA:SecurePassword;};encrypt=true
```

Reply:
referencing this documentation: https://learn.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver16#escaping-values-in-the-connection-url

It looks like this is possible! The example would look something like this 
```
sqlserver://HOST:PORT;database=DATABASE;user={MyServer/MyUser};password={ThisIsA:SecurePassword;};encrypt=true
```